decompy/RL/ActionGenerator/ActionGenerator.py

```diff
@@ -9,14 +9,14 @@ class ActionGenerator:
     llvm state it received in Step 1.
     """
 
-    ol = OptimizationLister.OptimizationLister()
-    cs = CodeSearcher.CodeSearcher() # TODO: add trie
+    trie = TrieGenerator().generates_trie()
+    ol = OptimizationLister()
+    cs = CodeSearcher(trie)  # TODO: add trie?
```

Comment:
please remove TODO comments. They are annoying because I have things to detect them lol (also it's done, so it shouldn't be there).